client/my-sites/resume-editing/style.scss

```diff
@@ -33,9 +34,20 @@
 }
 
 .resume-editing__post-title {
+	position: relative;
+	overflow: hidden;
 	color: $white;
 	display: block;
+	max-width: 25vw;
+	white-space: nowrap;
 	font-family: $serif;
 	font-size: 14px;
 	line-height: 1;
+
+	&::after {
+		@include long-content-fade( $size: 24px, $color: $blue-wordpress );
+		left: 25vw;
+		right: auto;
+		margin-left: -24px;
```

Comment:
>Interesting technique here.  Out of curiosity, would there be an easy way to achieve the same effect with an element that didn't have a set `max-width`?

I had toyed around unsuccessfully with a few other ideas on how to achieve this, particularly around overflows/padding (e.g. `overflow: hidden; padding-right: 20px; margin-right: -20px;`) and percentage offsets (`left: 100%;`). I think I'd need to see a concrete example to have a better idea of whether this would be possible, as the need for a long content fade would seem to be predicated on the existence of some maximum width (whether on the element itself or a parent wrapper).